### PR TITLE
Disqus included in sub pages

### LINKED ISSUE #260: Flip-Flops missing discussion section
image

### PR DESCRIPTION
**Pull request by ShubhankarKG**
Fixes #260 
Changes :-
1. Subpages of combinational, flipflop, MSI, and registers have disqus now.
2. Added new line at EOF in .gitignore.
3. Resolved grammatical error in index.md.

## ✅️ By submitting this PR, I have verified the following

* [x] Checked to see if a similar PR has already been opened 🤔️
* [x] Reviewed the contributing guidelines 🔍️

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
Please review @YashKumarVerma

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,4 @@ node_modules
 .idea
 .idea/workspace.xml
 node_modules/
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
```
Comment (YashKumarVerma) [inline .gitignore:10]: this was part of a previous PR.
any particular reason this is still here ?

Comment (ShubhankarKG) [reply .gitignore:10]: I added the new line mentioned in the previous PR. Hence, the change was showed.

**Comment by ShubhankarKG** [comment]
Please review @YashKumarVerma

**Comment by YashKumarVerma** [comment]
Good work !

**Pull request merged**